crates/ty_ide/src/hover.rs

```diff
@@ -93,13 +93,13 @@ impl<'a, 'db> IntoIterator for &'a Hover<'db> {
     }
 }
 
-pub struct DisplayHover<'a> {
-    db: &'a dyn Db,
-    hover: &'a Hover<'a>,
+pub struct DisplayHover<'db, 'a> {
+    db: &'db dyn Db,
+    hover: &'a Hover<'db>,
     kind: MarkupKind,
 }
```

Comment:
This is debris from versions where this was being asked to store `CallSignatureDetails`, which is actually invariant over `'db`. Leaving it like this just simplifies any potential future work.